expression/builtin_encryption_vec.go

```diff
@@ -67,9 +67,19 @@ func (b *builtinAesDecryptSig) vecEvalString(input *chunk.Chunk, result *chunk.C
 	isWarning := !b.ivRequired && len(b.args) == 3
 	isConstKey := b.args[1].ConstItem(b.ctx.GetSessionVars().StmtCtx)
 
-	var key []byte
```

Comment:
Have these two functions(`builtinAesDecryptSig` and `builtinAesDecryptIVSig`) been accidentally modified incorrectly?